src/pipelines.py

```diff
@@ -40,61 +37,48 @@ def unet(config, train_mode):
+def double_unet(config):
+    save_output = False
+    load_saved_output = False
+    preprocessing = preprocessing_inference(config)
 
-    mask_resize = Step(name='mask_resize',
-                       transformer=make_apply_transformer(resize_image,
-                                                          output_name='resized_images',
-                                                          apply_on=['images', 'target_sizes']),
-                       input_data=['input'],
-                       input_steps=[unet_multitask],
-                       adapter={'images': ([(unet_multitask.name, 'mask_prediction')]),
-                                'target_sizes': ([('input', 'target_sizes')]),
-                                },
+    unet_simple = Step(name='unet_simple',
+                       transformer=PyTorchUNet(**config.unet_simple),
+                       input_steps=[preprocessing],
+                       is_trainable=True,
                        cache_dirpath=config.env.cache_dirpath,
-                       save_output=save_output)
+                       save_output=save_output, load_saved_output=load_saved_output)
 
-    contour_resize = Step(name='contour_resize',
-                          transformer=make_apply_transformer(resize_image,
-                                                             output_name='resized_images',
-                                                             apply_on=['images', 'target_sizes']),
-                          input_data=['input'],
-                          input_steps=[unet_multitask],
-                          adapter={'images': ([(unet_multitask.name, 'contour_prediction')]),
-                                   'target_sizes': ([('input', 'target_sizes')]),
-                                   },
-                          cache_dirpath=config.env.cache_dirpath,
-                          save_output=save_output)
+    masks = mask_postprocessing_simple(unet_simple, config, save_output=save_output)
+
+    unet_borders = Step(name='unet_borders',
+                        transformer=PyTorchUNet(**config.unet_borders),
+                        input_steps=[preprocessing],
+                        is_trainable=True,
+                        cache_dirpath=config.env.cache_dirpath,
+                        save_output=save_output, load_saved_output=load_saved_output)
 
-    detached = Step(name='detached',
-                    transformer=Postprocessor(),
-                    input_steps=[mask_resize, contour_resize],
-                    adapter={'images': ([(mask_resize.name, 'resized_images')]),
-                             'contours': ([(contour_resize.name, 'resized_images')]),
-                             },
-                    cache_dirpath=config.env.cache_dirpath,
-                    save_output=save_output)
+    borders = mask_postprocessing_borders(unet_borders, config, save_output=save_output)
+
+    watersheder = Step(name='watershed',
```

Comment:
I love the name watersheder. Sounds like a superhero! 
"Is it a bird is it a morphological closing transform. No it's a watersheder"